src/batchjudge.cpp

```diff
@@ -52,9 +52,9 @@ int BatchJudge(int pid, int td, int boxid, int tl, int ml, int ol, int testee,
 
   // invoke box command
   if (lang == "python2") {
-    sandboxExec(boxid, opt, {"/usr/bin/env", "python2.7", "main.pyc"});
+    sandboxExec(boxid, opt, {"/usr/bin/env", "python2", "main.pyc"});
   } else if (lang == "python3") {
-    sandboxExec(boxid, opt, {"/usr/bin/env", "python3.7", "main.pyc"});
+    sandboxExec(boxid, opt, {"/usr/bin/env", "python3", "main.pyc"});
```

Comment:
I think something like
```cpp
if (lang.startswith("python")) {
  sandboxExec(..., {"/usr/env/bin", lang, ...});
} else {
  // C++
}
```

Reply:
How about something like this?  
use `lang.find_first_of("python") == 0` as `lang.starts_with`.

Maybe we could change this when C++20 being released officially.

Reply:
LGTM